windows-driver-docs-pr/install/combining-platform-extensions-with-operating-system-versions.md

```diff
@@ -99,6 +99,25 @@ When you install a driver package on a device, Windows follows these steps to pr
 
 1. If Windows cannot find a matching [**INF *Models* section**](inf-models-section.md), it will not install that driver package on the device.
 
+### How *TargetOsVersion* decorations are resolved
+
+When determining the applicable [**INF *Models* sections**](inf-models-section.md), the evaluation is performed per-line within the [**INF Manufacturer section**](inf-manufacturer-section.md), where, for a given OS version, each line may have up to one applicable section.
+
+The below INF excerpt shows an example where [ExampleModelsSection_1.NTamd64.10.0...17134] will be selected on a Windows 10 build equal to or greater than 17134, and *both* [ExampleModelsSection_1.NTamd64.10.0...17134] and [ExampleModelsSection_2.NTamd64.10.0...22000] will be selected on a Windows 10 build equal to or greater than 22000.
```

Comment:
```suggestion
The below INF excerpt shows an example where [ExampleModelsSection_1.NTamd64.10.0...17134] is selected starting in Windows 10, version 1803 , and *both* [ExampleModelsSection_1.NTamd64.10.0...17134] and [ExampleModelsSection_2.NTamd64.10.0...22000] are selected starting in Windows 11, version 21H2.
```

Reply:
This logic doesn't necessarily tie to a released windows build (could be an insider or internal build, etc.), it might be a bit more direct to just say that you put in a build number and that's the build number it will be selected with